app/bundles/CampaignBundle/Form/Type/CampaignListType.php

```diff
@@ -70,28 +72,29 @@ public function configureOptions(OptionsResolver $resolver)
                         $choices = ['this' => $options['this_translation']] + $choices;
                     }
 
-                    return $choices;
+                    return array_flip($choices);
```

Comment:
It would be more performant to flip the key and value in the foreach above when the `$choices` array is being filled then to flip them afterwards.